ocs_ci/utility/uninstall_openshift_logging.py

```diff
@@ -29,6 +29,45 @@ def check_pod_vanished(pod_names):
             raise UnexpectedBehaviour
 
 
+def delete_logging_namespaces(force=False):
+    """
+    Deleting namespaces
+    1. Openshift-operators-redhat
+    2. Openshift-logging
+
+    """
+    openshift_logging_namespace = ocp.OCP(
+        kind=constants.NAMESPACES, namespace=constants.OPENSHIFT_LOGGING_NAMESPACE
+    )
+    openshift_operators_redhat_namespace = ocp.OCP(
+        kind=constants.NAMESPACES,
+        namespace=constants.OPENSHIFT_OPERATORS_REDHAT_NAMESPACE,
+    )
+
+    if openshift_operators_redhat_namespace.get():
+        try:
+            assert openshift_operators_redhat_namespace.delete(
+                resource_name=constants.OPENSHIFT_OPERATORS_REDHAT_NAMESPACE,
+                force=force,
+            )
+            logger.info(
+                "The project openshift-opertors-redhat got deleted successfully"
+            )
+        except CommandFailed as e:
+            assert "not found" in str(e)
+            logger.info("Namespace not found" f"Error message {e}")
+
+    if openshift_logging_namespace.get():
+        try:
+            assert openshift_logging_namespace.delete(
```

Comment:
same as above comment, 
ocp.delete throws an exception incase the command exection fails, do we still need to throw an assert?